schema/test/unit/trigger_functions/commit_form_change_test.sql

```diff
@@ -163,6 +163,29 @@ select lives_ok(
   'Function does not throw an exception when trying to commit a change with an empty new_form_data'
 );
 
+-- setting up pending change without specifying the record id
+delete from mock_schema.mock_form_change;
+delete from mock_schema.mock_table;
+insert into mock_schema.mock_form_change(new_form_data, operation, form_data_schema_name, form_data_table_name, change_status)
+values (
+  '{"textCol":"test text", "intCol":234, "bool_col": true, "requiredCol": "req", "defaultedCol": 1}',
+  'INSERT', 'mock_schema', 'mock_table', 'test_pending'
+);
+
+update mock_schema.mock_form_change set change_status = 'test_committed';
+
+-- it should set the form_data_record_id using a sequence on the form_data_table_name id column if not provided
+
+select is(
+  (select count(*) from mock_schema.mock_table where text_col='test text'),
+  1::bigint,
+  'A record should be inserted when a pending change is committed'
+);
+
+select ok(
+  ((select form_data_record_id from mock_schema.mock_form_change) is not null),
+  'form_data_record_id should be set when committing a change without specifying the record id'
+);
```

Comment:
should we check that the `form_data_record_id` is set to the current sequence value? Or maybe just to the ID that was just inserted? I guess they'll just be `1` all around for the sake of this test

```suggestion
select is(
  (select form_data_record_id from mock_schema.mock_form_change),
  (select currval(pg_get_serial_sequence('mock_schema.mock_table', 'id'))),
  'form_data_record_id should be set when committing a change without specifying the record id'
);
```

Reply:
I can change that. I think it's 4 actually, because other records were inserted in the previous tests